### PR TITLE
[AutoRound] AutoRound for llama3-8b-instruct-chat

### PR DESCRIPTION
**Pull request by Zhenzhong1**
## Type of Change

New features.

## Description

* Remove convert_llama3.py
* AutoRound for llama3-8b-instruct-chat inference pass with the correcy accuracy.

## Expected Behavior & Potential Risk

N/A

## How has this PR been tested?

image

## Dependency Change?

N/A

### CODE REVIEW

File: neural_speed/convert/convert_quantized_llama.py [hunk 1]
```diff
@@ -109,105 +117,232 @@ def main(args_in: Optional[List[str]] = None) -> None:
...
+                text = text.encode("utf-8")
+            f.write(struct.pack("i", len(text)))
+            f.write(text)
+            f.write(struct.pack("f", score))
+
+        # 3. write tensors
+        list_vars = model
+        convert_to_fp32_tensor("model.embed_tokens.weight", "tok_embeddings.weight", list_vars, f)
+        convert_to_fp32_tensor("model.norm.weight", "norm.weight", list_vars, f)
+        convert_to_q4_bestla_tensor(f"lm_head.weight", f"output.weight", list_vars, f, quantize_config, n_head)
```
Comment (a32543254) [inline neural_speed/convert/convert_quantized_llama.py:193]: what if we get a lm_head.weight dtype = fp16 llama3 model ?

Comment (Zhenzhong1) [reply neural_speed/convert/convert_quantized_llama.py:193]: Not sure whether there is a validated FP16->int4 conversion pipeline in the Neural Speed, but it shouldn't have too many problems with a little effort.

At least we can convert back fp16 -> fp32 -> int4 and do the same pipeline we already have.

Comment (Zhenzhong1) [reply neural_speed/convert/convert_quantized_llama.py:193]: Not sure whether there is a validated conversion pipeline, but should be ok.

**Review by intellinjun — APPROVED** [review]

**Review by a32543254 — APPROVED** [review]
LGTM

**Pull request merged**